web/api/public/app/[app_id]/index.ts

```diff
@@ -22,6 +22,16 @@ export async function GET(
     app_id,
   });
 
+  // We only accept requests through the distribution origin
+  if (!isValidHostName(request)) {
+    return NextResponse.json(
+      {
+        error: "Invalid Request Origin, please use CDN",
```

Comment:
```suggestion
        error: "Please use https://world-id-assets.com for this endpoint",
```
or something along those lines

Reply:
using env value